src/router/index.js

```diff
@@ -1,38 +1,38 @@
 import Vue from 'vue';
 import Router from 'vue-router';
-import LandingPage from '../docs/LandingPage.vue';
-import ImageContainerDocs from '../docs/ImageContainerDocs.vue';
-import CardDocs from '../docs/CardDocs.vue';
-import TableDocs from '../docs/TableDocs.vue';
+import FgLandingPage from '../docs/FgLandingPage.vue';
+import FgImageContainerDocs from '../docs/FgImageContainerDocs.vue';
+import FgCardDocs from '../docs/FgCardDocs.vue';
+import FgTableDocs from '../docs/FgTableDocs.vue';
 
 Vue.use(Router);
 
 export default new Router({
   routes: [
     {
       path: '/',
-      name: 'LandingPage',
-      component: LandingPage,
+      name: 'FgLandingPage',
+      component: FgLandingPage,
     },
     {
       path: '/Introduction',
       name: 'Introduction',
-      component: LandingPage,
+      component: FgLandingPage,
     },
     {
-      path: '/ImageContainer',
-      name: 'Image Container',
-      component: ImageContainerDocs,
+      path: '/FgImageContainer',
+      name: 'Fg Image Container',
```

Comment:
Same goes for the links. Do we want prefix here or not, @zashton @logan-laughery - what do you prefer? 
```suggestion
      name: 'Image Container',
```

Reply:
I prefer not having the prefix here.